internal/charger/easee.go

```diff
@@ -24,6 +24,17 @@ type Easee struct {
 	cache         time.Duration
 }
 
+// charge mode definition
+const (
+	ModeOffline       int = 0
+	ModeDisconnected  int = 1
+	ModeAwaitingStart int = 2
+	ModeCharging      int = 3
+	ModeCompleted     int = 4
+	ModeError         int = 5
+	ModeReadyToCharge int = 6
+)
```

Comment:
@andig bin mir nicht sicher ob "the go ways" ist. Hatte zwischenzeitlich auch noch einen expliziten Typen (analog remote.go) drin. Das war dann in der Benutzung aber etwas sperrig.

Reply:
Statt den Folgedeklarationen int = x kannst Du auch iota schreiben- das inkrementiert immer weiter

Reply:
https://yourbasic.org/golang/iota/

Reply:
Ja, das hatte ich gesehen. Fan das autoincrement hier aber eher kontraproduktiv weil ich dann nicht mehr einfach (also ohne selbst zählen) erkennen kann was es bedeutet wenn in der response bspw Modus 5 steht. Daher hier die explizite Schreibweise.